lang/cs/Org.Apache.REEF.IMRU/OnREEF/Driver/ServiceAndContextConfigurationProvider.cs

```diff
@@ -166,9 +175,13 @@ internal ContextAndServiceConfiguration GetDataLoadingConfigurationForEvaluatorB
 
             var serviceConf =
                 TangFactory.GetTang()
-                    .NewConfigurationBuilder(ServiceConfiguration.ConfigurationModule.Build(),
+                    .NewConfigurationBuilder(
+                        ServiceConfiguration.ConfigurationModule
+                        .Set(ServiceConfiguration.Services, GenericType<TaskStateService>.Class)
```

Comment:
Any particular advantage for putting it as service as compared to context start handler? I remember @markusweimer  saying sometime ago that service APIs need some modifications.